content/en/docs/concepts/cluster-administration/certificates.md

```diff
@@ -25,7 +25,7 @@ manually through `easyrsa`, `openssl` or `cfssl`.
         tar xzf easy-rsa.tar.gz
         cd easy-rsa-master/easyrsa3
         ./easyrsa init-pki
-1.  Generate a CA. (`--batch` set automatic mode. `--req-cn` default CN to use.)
+1.  Generate a CA. (`--batch` set automatic mode. `--req-cn` default Common Name (CN) to use.)
```

Comment:
This line has nested brackets. Fine in code; odd in English text.
How about:
```suggestion
1.  Generate a new certificate authority (CA). `--batch` sets automatic mode; `--req-cn` specifies the Common Name (CN) for the CA's new root certificate.
```

Also consider adding a newline after “certificate authority (CA).”.

Reply:
Thanks for your suggestion.  I'll fix it.